### PR TITLE
Feature/migrate frontend

### PR DESCRIPTION
**Pull request by BeatriceDuncan**
### Notion card link(s)

- [Migrate frontend from Realtime Database to Firestore](https://www.notion.so/normative/Migrate-frontend-from-Realtime-Database-to-Firestore-768929a8a44b4fdca2aae6fc9fd1cfed)
- [React: Add accountable organization contact in place page](https://www.notion.so/normative/React-Add-accountable-organization-contact-in-place-page-fc2ad1a5861e489fafe30bcfe4553c9a)

### Description (Proposed changes, additional context, chosen approach/alternative approaches, gotchas, etc)

- Updates firebase.ts to call new dtpr-canada firestore. It does not yet call dtpr-usa firestore.
- Inside of these new query functions the data is manipulated to replicate legacy data.
- Type interfaces & mockData is updated to reflect updated data structure.
- Data is queried via customerId which is saved as an environment variable.
- This PR also adds the accountable contact details to the place view.

### Steps for reviewer(s) to reproduce/test

- Replace local .env file with updated firebase config vars found in [Heroku Review Apps](https://dashboard.heroku.com/pipelines/bec9174c-86b3-4c82-b710-60fe9b7d324f/settings) or in the [Slack](https://normative.slack.com/archives/C01G91SGX6U/p1634227866226700?thread_ts=1634227847.226600&cid=C01G91SGX6U) channel.
- The base url for Heroku Review App url is https://dtpr-sticker-feature-mi-odgb2k.herokuapp.com
- Check views by appending place & sensor ids to the base url e.g.
Place = Boston:     https://dtpr-sticker-feature-mi-odgb2k.herokuapp.com/places/lK7pio6ro
Place = Normative:      https://dtpr-sticker-feature-mi-odgb2k.herokuapp.com/places/Ur-FM0fJw
A legacy sensor:      https://dtpr-sticker-feature-mi-odgb2k.herokuapp.com/sensors/-MVHOEpD1qFsmwdz7P-D
A new sensor added via retool:      https://dtpr-sticker-feature-mi-odgb2k.herokuapp.com/sensors/1iAQWQyJp50uWc8ACxJZ
- Try navigating to a URL for a place or sensor without / with a different customerId: this shouldn't work & you shouldn't get any data returned.

### CODE REVIEW

File: src/containers/Places/Place.tsx [hunk 1]
```diff
@@ -38,7 +38,11 @@ function Place() {
   useEffect(() => {
     if (place.data) {
       sensorsActions.onRequest();
-      getSensors(place.data, sensorsActions.onSuccess, sensorsActions.onError);
+      getSensorsInPlace(
```
Comment (tonybennett) [inline src/containers/Places/Place.tsx]: Bit of a nit (no need to change it) but this function gets sensors by place not in place

**Review by tonybennett — COMMENTED** [review]
Looks good overall

- On `develop`, all tests pass running `yarn test`. On this branch the tests do not pass. So either the tests should be removed if we aren't using them anymore, or fixed.

File: src/containers/Places/Place.tsx (hunk 1 above)
Comment (BeatriceDuncan) [reply src/containers/Places/Place.tsx]: Done!

**Comment by BeatriceDuncan** [comment]
>     * On `develop`, all tests pass running `yarn test`. On this branch the tests do not pass. So either the tests should be removed if we aren't using them anymore, or fixed.

This code has been commented out for now and fixing testing has been flagged in [Notion](https://www.notion.so/normative/Chore-address-testing-errors-0d85f040ff0b4a528e764f150148627d).

**Review by tonybennett — APPROVED** [review]
Not sure if @turnKei wants to give the final thumbs up but it's good from a code perspective

**Pull request merged**